sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java

```diff
@@ -1026,11 +1026,29 @@ public boolean isMapCollapsed() {
 
     /**
      * Sets highlights on a list of indoor map entities to the specified ARGB color.
+     *
+     * @param indoorMapId ID for the indoor map the entities are in.
```

Comment:
UUID

Reply:
It's mentioned as simply id in rest of the doc comments. Should I update them all to UUID for consistency? 
https://www.wrld3d.com/android/latest/docs/api/EegeoMap/

Reply:
OK leave it if it's more consistent with elsewhere